frontend/public/components/machine-set.tsx

```diff
@@ -35,15 +34,19 @@ const machineReplicasModal = (
   configureReplicaCountModal({
     resourceKind,
     resource,
-    title: 'Edit Machine Count',
-    message: `${resourceKind.labelPlural} maintain the proper number of healthy machines.`,
+    // t('machine-sets~Edit Machine Count')
+    titleKey: 'machine-sets~Edit Machine Count',
```

Comment:
The labels here and below should be sentence case

```suggestion
    titleKey: 'machine-sets~Edit machine count',
```